[2024-01-11T17:53:29.026+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: rds_test_2.send_slack manual__2024-01-11T17:53:18.260680+00:00 [queued]>
[2024-01-11T17:53:29.059+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: rds_test_2.send_slack manual__2024-01-11T17:53:18.260680+00:00 [queued]>
[2024-01-11T17:53:29.060+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-01-11T17:53:29.063+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2024-01-11T17:53:29.064+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-01-11T17:53:29.115+0000] {taskinstance.py:1300} INFO - Executing <Task(SlackWebhookOperator): send_slack> on 2024-01-11 17:53:18.260680+00:00
[2024-01-11T17:53:29.123+0000] {standard_task_runner.py:55} INFO - Started process 115 to run task
[2024-01-11T17:53:29.133+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'rds_test_2', 'send_slack', 'manual__2024-01-11T17:53:18.260680+00:00', '--job-id', '109', '--raw', '--subdir', 'DAGS_FOLDER/rds_test_2.py', '--cfg-path', '/tmp/tmpthr68ehp']
[2024-01-11T17:53:29.143+0000] {standard_task_runner.py:83} INFO - Job 109: Subtask send_slack
[2024-01-11T17:53:29.427+0000] {task_command.py:388} INFO - Running <TaskInstance: rds_test_2.send_slack manual__2024-01-11T17:53:18.260680+00:00 [running]> on host 3d6f932ce29c
[2024-01-11T17:53:29.566+0000] {abstractoperator.py:613} ERROR - Exception rendering Jinja template for task 'send_slack', field 'message'. Template: '```\n{{ result_str }}\n```'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/abstractoperator.py", line 606, in _do_render_template_fields
    seen_oids,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/abstractoperator.py", line 663, in render_template
    return render_template_to_string(template, context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 288, in render_template_to_string
    return render_template(template, cast(MutableMapping[str, Any], context), native=False)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 283, in render_template
    return "".join(nodes)
  File "<template>", line 13, in root
  File "/home/airflow/.local/lib/python3.7/site-packages/jinja2/runtime.py", line 852, in _fail_with_undefined_error
    raise self._undefined_exception(self._undefined_message)
jinja2.exceptions.UndefinedError: 'result_str' is undefined
[2024-01-11T17:53:29.585+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1374, in _run_raw_task
    self._execute_task_with_callbacks(context, test_mode)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1493, in _execute_task_with_callbacks
    task_orig = self.render_templates(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 2115, in render_templates
    original_task.render_template_fields(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 1179, in render_template_fields
    self._do_render_template_fields(self, self.template_fields, context, jinja_env, set())
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 75, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/abstractoperator.py", line 606, in _do_render_template_fields
    seen_oids,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/abstractoperator.py", line 663, in render_template
    return render_template_to_string(template, context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 288, in render_template_to_string
    return render_template(template, cast(MutableMapping[str, Any], context), native=False)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 283, in render_template
    return "".join(nodes)
  File "<template>", line 13, in root
  File "/home/airflow/.local/lib/python3.7/site-packages/jinja2/runtime.py", line 852, in _fail_with_undefined_error
    raise self._undefined_exception(self._undefined_message)
jinja2.exceptions.UndefinedError: 'result_str' is undefined
[2024-01-11T17:53:29.633+0000] {taskinstance.py:1323} INFO - Marking task as UP_FOR_RETRY. dag_id=rds_test_2, task_id=send_slack, execution_date=20240111T175318, start_date=20240111T175329, end_date=20240111T175329
[2024-01-11T17:53:29.703+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 109 for task send_slack ('result_str' is undefined; 115)
[2024-01-11T17:53:29.738+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-01-11T17:53:29.843+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
